Cannot build 'summoner-tui'

I'm trying to use `ghc-musl` to build `summoner-tui`. However, I see the following error when trying to do this:

```
Linking /mnt/dist-newstyle/build/x86_64-linux/ghc-8.8.1/summoner-tui-2.0.0.0/x/summon-tui/build/summon-tui/summon-tui ...
/nix/store/b1zj8ydpycg7vncj40mafr177bs66gw1-binutils-2.31.1/bin/ld: cannot find -ltinfo
collect2: error: ld returned 1 exit status
`cc' failed in phase `Linker'. (Exit code: 1)
cabal: Failed to build exe:summon-tui from summoner-tui-2.0.0.0.
```

--- a/README.md
+++ b/README.md
@@ -9,11 +9,11 @@ used libraries and build tools.
 
 Here are the latest images currently published in Docker Hub:
 
-* `utdemir/ghc-musl:v4-libgmp-ghc881`
-* `utdemir/ghc-musl:v4-integer-simple-ghc881`
-* `utdemir/ghc-musl:v4-libgmp-ghc865`
-* `utdemir/ghc-musl:v4-integer-simple-ghc865`
-* `utdemir/ghc-musl:v4-libgmp-ghc844`
+* `utdemir/ghc-musl:v5-libgmp-ghc881`
+* `utdemir/ghc-musl:v5-integer-simple-ghc881`
+* `utdemir/ghc-musl:v5-libgmp-ghc865`
+* `utdemir/ghc-musl:v5-integer-simple-ghc865`
+* `utdemir/ghc-musl:v5-libgmp-ghc844`
 
 ## Usage
 
@@ -27,7 +27,7 @@ inside the container:
 
 ```
 $ cd myproject/
-$ docker run -itv $(pwd):/mnt utdemir/ghc-musl:v4-libgmp-ghc881
+$ docker run -itv $(pwd):/mnt utdemir/ghc-musl:v5-libgmp-ghc881
 sh$ cd /mnt
 sh$ cabal new-update
 sh$ cabal new-build
@@ -41,7 +41,7 @@ host machine:
 ```
 docker:
   enable: true
-  image: utdemir/ghc-musl:v4-libgmp-ghc881
+  image: utdemir/ghc-musl:v5-libgmp-ghc881
 ```
 
 Make sure to pick an image with the GHC version compatible with the

--- a/default.nix
+++ b/default.nix
@@ -12,7 +12,7 @@ let
 user = "utdemir";
 name = "ghc-musl";
 tag = lib.concatStringsSep "-" [
-  "v4"
+  "v5"
   (if integer-simple then "integer-simple" else "libgmp")
   compiler
 ];

--- a/default.nix
+++ b/default.nix
@@ -33,6 +33,7 @@ libraries = with pkgsMusl; [
   musl
   zlib zlib.static
   libffi (libffi.override { stdenv = makeStaticLibraries stdenv; })
+  ncurses (ncurses.override { enableStatic = true; })
 ] ++ lib.optionals (!integer-simple) [ gmp (gmp.override { withStatic = true; }) ];
 
 packages = with pkgsMusl; [

